components/input/input.css

```diff
@@ -0,0 +1,55 @@
+@import url("/docs/main.css");
+
+.input_container {
+    padding: 2rem;
+    margin: 1rem 0 3rem 0;
+    border: 1px solid var(--light-grey-color);
+    box-shadow: 2px 2px 1px var(--shadow-color);
```

Comment:
you can make variable for shadow like this
--Shadow-01 : 2px 2px 1px var(--shadow-color);
and use Shadow-01 variable